src/systemd/sd-device.h

```diff
@@ -136,6 +136,7 @@ int sd_device_monitor_stop(sd_device_monitor *m);
 
 int sd_device_monitor_filter_add_match_subsystem_devtype(sd_device_monitor *m, const char *subsystem, const char *devtype);
 int sd_device_monitor_filter_add_match_tag(sd_device_monitor *m, const char *tag);
+int sd_device_monitor_filter_add_match_sysattr(sd_device_monitor *m, const char *sysattr, const char *value, int match);
```

Comment:
any chance we can add a unit test for this in src/libsystemd/sd-device/test-sd-device-monitor.c ?

Reply:
same for sd_device_monitor_filter_add_match_parent

Reply:
Done.